bench/bench_array1d.py

```diff
@@ -13,7 +13,9 @@ def runge_kutta_step(f, x0, dt, t=None):
     k2 = f(t, x0 + k1 / 2) * dt
     k3 = f(t, x0 + k2 / 2) * dt
     k4 = f(t, x0 + k3) * dt
-    x_new = x0 + (k1 + 2 * k2 + 2 * k3 + k4) / 6
+    # workaround for a pypy bug
+    #x_new = x0 + (k1 + 2 * k2 + 2 * k3 + k4) / 6
+    x_new = x0 + (k1 + (k2 * 2) + (k3 * 2) + k4) / 6
```

Comment:
It would be good to add to the comment something about what the PyPy bug is so that we can know when to remove the strange brackets and why they are needed.

Reply:
The workaround is not about the parenthesis but about the order: `k2 * 2` works but not `2 * k2`. I added a reference to https://foss.heptapod.net/pypy/pypy/-/issues/3509 and remove the unnecessary parenthesis.